cime_config/machines/Depends.summit-gpu.cmake

```diff
@@ -0,0 +1,7 @@
+list(APPEND NOOPT_FILES
+  eam/src/dynamics/eul/dyn_comp.F90
```

Comment:
This is the original configuration, I guess this is not compiling time issue, these files are small, it could be not necessary to use optimization flags